src/bp-core/admin/js/moderation-page.js

```diff
@@ -56,10 +57,15 @@ jQuery( document ).ready( function ( $ ) {
 						}
 					}
 				}
-				$( '.bp-moderation-ajax-msg p' ).text( result.message ).parent().removeClass( 'hidden' );
-				setTimeout( function () {
-					location.reload();
-				}, 2000 );
+
+                var url = window.location.href;
+                if (url.indexOf('?') > -1) {
+                    url += '&moderation_msg=' + result.message;
```

Comment:
@janakkaneriya007 Instead of passing string message as query argument we should pass integer value and depend on that value we should show the message.